src/library-authoring/data/apiHooks.ts

```diff
@@ -372,3 +394,18 @@ export const useRestoreCollection = (libraryId: string, collectionId: string) =>
     },
   });
 };
+
+/**
+ * Use this mutation to update collections related a component in a library
+ */
+export const useUpdateComponentCollections = (libraryId: string, usageKey: string) => {
+  const queryClient = useQueryClient();
+  return useMutation({
+    mutationFn: async (collectionKeys: string[]) => updateComponentCollections(usageKey, collectionKeys),
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    onSettled: (_data, _error, _variables) => {
```

Comment:
This params are not necessary

Reply:
Updated: https://github.com/openedx/frontend-app-authoring/pull/1373/commits/44c4c250ffdd6358b39cdcc0779f200b6bfeb8f5